vllm_hpu_extension/flags.py

```diff
@@ -8,12 +8,14 @@
 import os
 from packaging.version import Version
 from packaging.specifiers import SpecifierSet
-from functools import cache
 
 from vllm_hpu_extension.environment import get_environment
 from vllm_hpu_extension.kernels import fsdpa
 
 
+detected = None
```

Comment:
for me  it seems to be better to have this line just before  def enabled_flags()